test/Microsoft.ApplicationInsights.AspNetCore.Tests/Extensions/ApplicationInsightsExtensionsTests.cs

```diff
@@ -142,6 +147,7 @@ public static void RegistersTelemetryConfigurationFactoryMethodThatReadsDevelope
             }
 
             [Fact]
+            [Trait("Category", "WindowsOnly")]
```

Comment:
why? Please comment

Reply:
Part of trial runs.. removed and fixed Linux failures .
(it was / vs \ :) )